src/main/scala/org/hammerlab/guacamole/loci/set/Contig.scala

```diff
@@ -14,7 +15,32 @@ import scala.collection.mutable.ArrayBuffer
 /**
  * A set of loci on a contig, stored/manipulated as loci ranges.
  */
-case class Contig(name: ContigName, private val rangeSet: RangeSet[JLong]) extends TruncatedToString {
+case class Contig(var name: ContigName, private var rangeSet: RangeSet[JLong]) extends TruncatedToString {
+
+  private def readObject(in: ObjectInputStream): Unit = {
```

Comment:
yea, weird semantics of Java `Serializable`, you make them `private` and not an `override`.